pkg/doc/did/doc.go

```diff
@@ -410,6 +440,36 @@ func populateServices(rawServices []map[string]interface{}) []Service {
 	return services
 }
 
+func populateRecipientKeys(recipientKeys []string, base string) []string {
```

Comment:
Maybe create one method `populateKeys()` and use it for the recipient and routing keys?

Reply:
done